crates/lib3h/src/engine/real_engine.rs

```diff
@@ -201,13 +199,13 @@ impl<T: Transport, D: Dht> Drop for RealEngine<T, D> {
 }
 
 /// Private
-impl<T: Transport, D: Dht> RealEngine<T, D> {
+impl<'engine, D: Dht> RealEngine<'engine, D> {
     /// Called on drop.
     /// Close all connections gracefully
     fn shutdown(&mut self) -> Lib3hResult<()> {
         let mut result = Ok(());
```

Comment:
What do you plan to do with this `result` afterwards ? :thinking:

Reply:
hah, good catch, that was supposed to be returned. Fixed :+1: